packages/console/src/pages/EnterpriseSsoDetails/index.tsx

```diff
@@ -95,11 +94,17 @@ function EnterpriseSsoConnectorDetails<T extends SsoProviderName>() {
       setIsDeleted(true);
 
       toast.success(t('enterprise_sso_details.enterprise_sso_deleted'));
-      navigate(enterpriseSsoPathname);
+      await mutateGlobal(
+        buildUrl('api/sso-connectors', {
+          page: '1',
```

Comment:
Is this intentionally switched to the first page?

Reply:
Good call, removed.